app/src/main/java/module2/globant/calculator/mvp/presenter/CalculatorPresenter.java

```diff
@@ -0,0 +1,80 @@
+package module2.globant.calculator.mvp.presenter;
+
+import android.app.Activity;
+
+import module2.globant.calculator.R;
+import module2.globant.calculator.Utils;
+import module2.globant.calculator.bus.RxBus;
+import module2.globant.calculator.bus.observers.OperationButtonBusObserver;
+import module2.globant.calculator.bus.observers.ResultButtonBusObserver;
+import module2.globant.calculator.mvp.model.CalculatorModel;
+import module2.globant.calculator.mvp.view.CalculatorView;
+
+public class CalculatorPresenter {
+
+    private CalculatorModel model;
+    private CalculatorView view;
+
+    public CalculatorPresenter(CalculatorModel model, CalculatorView view) {
+        this.model = model;
+        this.view = view;
+    }
+
+
+    public void onResultButtonPressed() {
+        if (validated()) {
+            view.setResultLabel(String.valueOf(model.doOperation(model.getOperation(), Double.parseDouble(view.getFieldOne()), Double.parseDouble(view.getFieldTwo()))));
+        }
+    }
+
```

Comment:
if you are adding double line break between methods, be consistent in all methods